packages/frontend/model/extract-structured-data.ts

```diff
@@ -0,0 +1,5 @@
+import { getObject } from '@frontend/model/validators';
+
+export function extractStructuredData(data: object): object {
```

Comment:
Could we rename this function `extract` for consistency with other the other `extract-` functions?

Also, would it be possible to create an interface for the return type here called `StructuredDataType` in [`index.d.ts`](https://github.com/guardian/dotcom-rendering/blob/master/packages/frontend/index.d.ts)?

Reply:
Happy to re-instate it though if you'd prefer. It's ready, in the git history :)

Reply:
Yeah, I take your point about defining an interface. We have accepted there will be a certain amount of duplication between types defined here and in frontend. But you're right, an interface for structured data wouldn't serve any purpose since we're just `JSON.stringify`-ing the object anyway.